src/components/Header/index.tsx

```
@@ -1,9 +1,11 @@
-'use client';
+"use client";
 
 import Link from "next/link";
 import { MoonIcon } from "@/icons/MoonIcon";
 import { GithubIcon } from "@/icons/GithubIcon";
 import { LinkedinIcon } from "@/icons/LinkedinIcon";
+import { usePathname } from "next/navigation";
+import path from "path";
```

ESLint

[error] 'path' is defined but never used.

(@typescript-eslint/no-unused-vars)